python/usb_console/simulation.py

```diff
@@ -87,28 +86,34 @@ def run(self):
         while step < num_steps and self.running:
             self.sim_time = self.main_state['follower']['dynamics']['time']
 
-            # Get sensor readings
+            # Step 1. Get sensor readings from simulation
             self.sensor_readings_follower = self.eng.sensor_reading(self.main_state['follower'],self.main_state['leader'], nargout=1)
             self.sensor_readings_leader = self.eng.sensor_reading(self.main_state['leader'],self.main_state['follower'], nargout=1)
-            # update dynamics
+            
+            # Step 2. Update dynamics
             self.main_state = self.eng.main_state_update(self.main_state, nargout=1)
-            # simulate flight computers
+            
+            # Step 3. Simulate flight computers
+            # Step 3.1. Use MATLAB simulation as a base
```

Comment:
By "base" I mean that some of the actuator commands come from the simulation itself, when the Flight Controller hasn't yet implemented a certain control. Gradually, calling update_FC_state will become unnecessary here once the FlightController is fully autonomous.

Reply:
you can also use `function actuator_commands = actuators_off_command()` to generate all the commands that will turn off the actuators as a base.